index.html

```diff
@@ -91,39 +91,28 @@
           </p>
           <p>
             Firecracker is used by/integrated with (in alphabetical order):
-            <a href="https://appfleet.com/">appfleet</a>
-            , containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            ,
-            <a href="https://fly.io">Fly.io</a>
-            ,
+            <a href="https://appfleet.com/">appfleet</a>,
+            containerd via
+            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>,
+            <a href="https://fly.io">Fly.io</a>,
             <a
```

Comment:
not: we can align this line to the previous style too with everything in one line

`<a href="https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support">Kata Containers</a>`